docs/reference/snapshot-restore/repository-s3.asciidoc

```diff
@@ -143,7 +143,8 @@ settings belong in the `elasticsearch.yml` file.
 
 `read_timeout`::
 
-    The socket timeout for connecting to S3. The value should specify the unit.
+    The maximum time for data to be transferred over an established, open connection
```

Comment:
To me this suggests that it's a timeout on the read of a whole blob, whereas in fact it's a timeout for the individual `read()` calls, each of which may only read a handful of bytes.

Reply:
What do you think about? 
```
The maximum time for data to be read from an established, open connection before the connection times out and is closed.
```

Reply:
Hm this is tricky, I've tried a few different ideas and am finding it hard to get rid of the ambiguity in a single sentence. This is about the best I came up with:

    The maximum time {es} will wait to receive the next byte of data over an established, open connection to the repository before it closes the connection.

If not that, I think we need several sentences.

Reply:
I like that, thanks for the suggestion!